src/common/snippets/include/snippets/lowered/loop_manager.hpp

```diff
@@ -85,6 +85,10 @@ class LinearIR::LoopManager {
         void set_outer_splited_loop(bool outer_splited_loop);
         void set_first_iter_handler(FirstIterHandler handler);
 
+        // Update the parameters of existing LoopPorts
+        void update_entry_points(const std::function<void(LoopPort&)>& updater);
+        void update_exit_points(const std::function<void(LoopPort&)>& updater);
+
```

Comment:
Minor:
Since we would like to leave only relevant changes (not include code style and not related changes to the fix), we have to not include these changes. Because looks like that these changes and changes in `InitLoops` are not relevant to the fix `iterator comparison from different lists` - and can be removed from the PR.

Reply:
The same comment for helper `is_loop_id_found` - it's not used in fix-related changes

Reply:
Code style changes is removed for simplicity on check.
`update_entry_points`, `update_exit_points `and `is_loop_id_found ` changes is excluded.
thanks Alexandra!